nixos/doc/manual/release-notes/rl-2105.xml

```diff
@@ -984,6 +984,24 @@ environment.systemPackages = [
      PostgreSQL 9.5 is scheduled EOL during the 21.05 life cycle and has been removed.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     <link xlink:href="https://www.xfce.org/">Xfce4</link> relies on
+     GIO/GVfs for userspace virtual filesystem access in applications
+     like <link xlink:href="https://docs.xfce.org/xfce/thunar/">thunar</link> and
+     <link xlink:href="https://docs.xfce.org/apps/gigolo/">gigolo</link>.
+     For that to work, the gvfs nixos service is enabled by default,
+     and it can be configured with the specific package that provides
+     GVfs. Until now Xfce4 was setting it to use a lighter version of
+     GVfs (without support for samba). To avoid conflicts with other
+     desktop environments this setting has been dropped. Users that
+     still want it should add the following to their system
+     configuration:
+     <programlisting>
+       <xref linkend="opt-services.gvfs.package" /> = pkgs.gvfs.override { samba = null; };
```

Comment:
I think setting the enable option to true is missing.
```suggestion
       <xref linkend="opt-services.gvfs.enable" /> = true;
       /* The following is only necessary to use a slimmed down version of gvfs, without samba support */
       <xref linkend="opt-services.gvfs.package" /> = pkgs.gvfs.override { samba = null; };
```

Reply:
Xfce enables it by default according to the note.

Reply:
Yes, gvfs is enabled by default in the xfce module.

Reply:
Oh sorry I had misread. Please ignore.